### PR TITLE
KPI Integration

### PR DESCRIPTION
**Pull request by jeafreezy**
### What does this PR do ?

This PR integrates the KPI endpoint on the landing page and includes some small fixes like typos.

### How to test ?

Visit: https://f-a-ir-emmanuels-projects-1886bda9.vercel.app

### CODE REVIEW

File: frontend/src/components/landing/kpi/kpi.tsx [hunk 1]
```diff
@@ -1,41 +1,68 @@
...
+          <p>{(error as Error)?.message || 'An unknown error occurred.'}</p>
+        </div>
+      </section>
+    );
+  }
+
+  const KPIs: TKPIS = [
+    {
+      figure: data?.total_models_published ?? 0,
+      label: APP_CONTENT.homepage.kpi.publishedAIModels,
```
Comment (omranlm) [inline frontend/src/components/landing/kpi/kpi.tsx:43]: we can add some format for the numbers

**Review by omranlm — APPROVED** [review]
Great and quick

File: frontend/src/components/landing/kpi/kpi.tsx (hunk 1 above)
Comment (jeafreezy) [reply frontend/src/components/landing/kpi/kpi.tsx:43]: Do you mean like 12k, 20k etc @omranlm ?

Comment (omranlm) [reply frontend/src/components/landing/kpi/kpi.tsx:43]: like 1,546 instead of 1546... no worries, I will do that with other text changes ...

Comment (jeafreezy) [reply frontend/src/components/landing/kpi/kpi.tsx:43]: Oh, okay. Thanks!

**Pull request merged**